### PR TITLE
Integration tests prep

### PR DESCRIPTION
**Pull request by adaszko**
* https://rchain.atlassian.net/browse/CORE-1273

Preparations needed before switching over from p2p tests to integration tests completely.

### CODE REVIEW

File: scripts/ci/run-integration-tests.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#/bin/bash -ue
+
+cd /tmp/rchain/integration-testing
+python3 -m pip install pipenv
```
Comment (odeac) [inline scripts/ci/run-integration-tests.sh]: this should go in integration-testing/install_dependencies.sh and here you should call that script. This way people can run integration-tests outside CI

Comment (adaszko) [reply scripts/ci/run-integration-tests.sh]: install_dependencies.sh is no more

Comment (odeac) [reply scripts/ci/run-integration-tests.sh]: then what do I need to do in order to run the tests locally?

File: integration-testing/Pipfile [hunk 2]
```diff
@@ -9,6 +9,3 @@ pytest = "*"
 delayed-assert = "*"
 
 [dev-packages]
-
-[requires]
-python_version = "3.7"
```
Comment (woky) [inline integration-testing/Pipfile]: Don't you _require_ it for `f'Hello: {name_var}'`?

**Pull request merged**